contrib/gce/configure.sh

```diff
@@ -104,6 +104,26 @@ if [ -f "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}" ]; then
   source "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}"
 fi
 
+# cgroupv2 setup
+if [ "${CONTAINERD_CGROUPV2:-"false"}"  == "true" ]; then
```

Comment:
This seems node e2e and COS specific. Can you move this into the COS node e2e specific init script? https://github.com/containerd/containerd/blob/master/test/e2e_node/gci-init.sh

That would limit the scope and impact of this change.

Reply:
Thanks for the feedback. Moved to this specific init script.

Reply:
From where is gci-init.sh invoked/being used today?

Reply:
It is used in metadata, similar to container-env file.  For example: https://github.com/kubernetes/test-infra/pull/22130/commits/8a193964a1fd038ed257cf5599b9d98ab41ad112#diff-83d41adf22b2d07de4221c2b714ee8fce0be04a51bb47c50a1a006f0b24c1216

Reply:
got it, makes sense. As discussed it is evaluated at the end of configure.sh here -- https://github.com/containerd/containerd/blob/master/contrib/gce/configure.sh#L226